internal/fourslash/fourslash.go

```diff
@@ -312,11 +317,89 @@ func (f *FourslashTest) GoToEOF(t *testing.T) {
 	f.goToPosition(t, LSPPos)
 }
 
+func (f *FourslashTest) GoToBOF(t *testing.T) {
+	f.goToPosition(t, lsproto.Position{Line: 0, Character: 0})
+}
+
+func (f *FourslashTest) GoToPosition(t *testing.T, position int) {
+	script := f.getScriptInfo(f.activeFilename)
+	LSPPos := f.converters.PositionToLineAndCharacter(script, core.TextPos(position))
+	f.goToPosition(t, LSPPos)
+}
+
 func (f *FourslashTest) goToPosition(t *testing.T, position lsproto.Position) {
 	f.currentCaretPosition = position
 	f.selectionEnd = nil
 }
 
+func (f *FourslashTest) GoToEachMarker(t *testing.T, markerNames []string, action func(t *testing.T, marker *Marker, index int)) {
```

Comment:
Out of curiosity, why is `t` passed through to the callback? The only test I see using this closes over the outer `FourslashTest` anyway (though who knows if that gets inlined).